front-end/src/domains/JobInfoItem/JobInfoItem.tsx

```diff
@@ -0,0 +1,134 @@
+import { useHistory } from "react-router-dom";
+import { AxiosError } from "axios";
+import { useBoolean, useFetch } from "../../hooks";
+import { formatDate, addHours } from "../../utils";
+import { Alert, Button, CalendarIcon, Confirm, Text } from "../../components";
+import { StyledJobInfoItem } from "./JobInfoItem.styled";
+import { API_ENDPOINT, PATH } from "../../constants";
+import { JobViewResponse } from "../../types";
```

Comment:
```suggestion
import { useHistory } from "react-router-dom";
import { AxiosError } from "axios";
import { formatDate, addHours } from "../../utils";
import { useBoolean, useFetch } from "../../hooks";
import { Alert, Button, CalendarIcon, Confirm, Text } from "../../components";
import { StyledJobInfoItem } from "./JobInfoItem.styled";
import { API_ENDPOINT, PATH } from "../../constants";
import { JobViewResponse } from "../../types";
```


전에 import 순서 정할때 utils -> hooks 였던거 같은데 맞나요?

어디다가 글로 적어둔거 같은데 못찾겠네요..